plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -53,6 +54,13 @@ export class EventsProcessor {
         eventUuid: string
     ): Promise<PreIngestionEvent | null> {
         if (!UUID.validateString(eventUuid, false)) {
+            void captureWarning(
```

Comment:
We should _really_ avoid voiding promises due to the amount of issues they cause.

Take a look at `pluginsServer.promiseManager`